### PR TITLE
[CI] Change folder structure.

### PR DESCRIPTION
**Pull request by ohsewon**
change folder name to use config files by copying instead of linking.

Signed-off-by: Sewon Oh sewon.oh@samsung.com

---
# [Template] PR Description

In general, github system automatically copies your commit message for your convenience.
Please remove unused part of the template after writing your own PR description with this template.
```bash
$ git commit -s filename1 filename2 ... [enter]

Summarize changes in around 50 characters or less

More detailed explanatory text, if necessary. Wrap it to about 72
characters or so. In some contexts, the first line is treated as the
subject of the commit and the rest of the text as the body. The
blank line separating the summary from the body is critical;
various tools like `log`, `shortlog` and `rebase` can get confused 
if you run the two together.

Further paragraphs come after blank lines.

**Changes proposed in this PR:**
- Bullet points are okay, too
- Typically a hyphen or asterisk is used for the bullet, preceded
  by a single space, with blank lines in between, but conventions vary here.

Resolves: #123
See also: #456, #789

**Self evaluation:**
1. Build test: [ ]Passed [ ]Failed [*]Skipped
2. Run test: [ ]Passed [ ]Failed [* ]Skipped

**How to evaluate:**
1. Describe how to evaluate in order to be reproduced by reviewer(s).

Add signed-off message automatically by running **$git commit -s ...** command.

$ git push origin <your_branch_name>
```

### CODE REVIEW

**Comment by myungjoo** [comment]
@leemgs Are you really going to set /Documentation/ci-nnstreamer as the default ci config path?

**Comment by leemgs** [comment]
> Are you really going to set /Documentation/ci-nnstreamer as the default ci config path?

Yes. Let's use `/Documentation/ci-nnstreamer/` as the default directory to archive the ci config files for NNStreamer CI system. Maybe do you want to /ci-nnstreamer instead of /Documentation/ci-nnstreamer ? It is also not bad.

File: Documentation/ci-nnstreamer/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+These are TAOS-CI files for nnstreamer.
+
+- http://suprem.sec.samsung.net/STAR/TAOS-CI
+
+## How to use
+
+After adding TAOS-CI submodule in your project, copy and overwrite them.
+
+```bash
+$ yes | cp -rf $PWD/Documentation/taos/* $PWD/TAOS-CI/ci/taos/
```
Comment (leemgs) [inline Documentation/ci-nnstreamer/README.md]: I think that you keep "yes" command to handle the existing files  of  ./ci/taos/***.  Is it right? Looks good.

**Comment by myungjoo** [comment]
> Yes. Let's use /Documentation/ci-nnstreamer/ as the default directory to archive the ci config files for NNStreamer CI system. Maybe do you want to /ci-nnstreamer instead of /Documentation/ci-nnstreamer ? It is also not bad.

I was asking it because having ```nnstreamer``` string in ci config path appears strange to me.

**Comment by ohsewon** [comment]
close cause of apptest ci.

**Pull request closed without merging**